tabular/src/autogluon/tabular/task/tabular_prediction/predictor.py

```diff
@@ -1087,6 +1091,47 @@ def distill(self, train_data=None, tuning_data=None, augmentation_data=None, tim
                                      verbosity=verbosity, models_name_suffix=models_name_suffix, teacher_preds=teacher_preds,
                                      augmentation_data=augmentation_data, augment_method=augment_method, augment_args=augment_args)
 
+    def plot_ensemble_model(self):
+        """
+            Output the visualized stack ensemble architecture of a model trained by `fit()`. 
+            The plot is stored to a file, `ensemble_model.png` in folder `Predictor.output_directory` 
+
+            This function requires `graphviz` and `pygraphviz` to be installed because this visualization depends on those package.
+            Unless this function will raise `ImportError` without being able to generate the visual of the ensemble model.
+
+        """
+        try:
+            import pygraphviz
```

Comment:
Would graphviz work instead of pygraphviz? What are the advantages of pygraphviz? I mention this because we already use graphviz elsewhere in the project.

Reply:
Dose graphviz provide Python front-end? If we have graphviz in the package already, then installing pygraphviz should be easily done by adding pip install pygraphviz.

Reply:
I meant the python package [graphviz](https://github.com/xflr6/graphviz) (separate from graphviz the non-python package).